cmd/start.go

```diff
@@ -128,7 +124,10 @@ func start(ctx context.Context) {
 
 								// Notify BHE instance of task start
 								currentTask = &executableTasks[0]
-								startTask(ctx, *bheInstance, bheClient, currentTask.Id)
+								if err := startTask(ctx, *bheInstance, bheClient, currentTask.Id); err != nil {
+									log.Error(err, "failed to start task")
+								}
```

Comment:
If we fail to start the task then we should probably unset currenttask and try again after the next heartbeat

Reply:
Yeah, I didn't know for sure what the best error state would be here, so just logged for now. More than happy to add this behavior while we're here